### PR TITLE
feat : added basic routing and pages

### PR DESCRIPTION
**Pull request by raman1999**

### CODE REVIEW

File: src/Routes/RoutingPath.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  Signup,
+  VideoListing,
+  LikedVideos,
+  History,
+  NotFound,
+  Playlist,
+  PlaylistDetails,
+  VideoDetails,
+  WatchLater,
+} from "../Pages";
```
Comment (therajatg) [inline src/Routes/RoutingPath.jsx:14]: won't from "../Pages/index,js" come here

**Review by therajatg — APPROVED** [review]
otherwise liked the way you built the basic structure first by making all the pages inside the pages folder, importing them in index.js.

**Review by RahulSingh9131 — APPROVED** [review]
I think in your RoutingPath.jsx you need to import components from your pages/index.jsx. Else everything is good.

**Comment by raman1999** [comment]
No, index.js is by default taken ,so ../Pages is also correct

**Pull request merged**